libs/shared/types/src/methodology/methodology-enum.types.ts

```diff
@@ -33,3 +33,14 @@ export enum MethodologyDocumentEventName {
 export enum MethodologyEvaluationResult {
   APPROVED = 'APPROVED',
 }
+
+export enum MethodologyDocumentEventAttributeFormat {
+  CUBIC_METER = 'CUBIC_METER',
+  DATE = 'DATE',
+  KILOGRAM = 'KILOGRAM',
+  LITRE = 'LITRE',
```

Comment:
We are using American spelling, not British
```suggestion
  LITER = 'LITER',
```